test/snippet/range/views/minimiser_hash.cpp

```diff
@@ -0,0 +1,21 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/range/views/minimiser_hash.hpp>
+
+using seqan3::operator""_dna4;
+using seqan3::operator""_shape;
+
+int main()
+{
+    std::vector<seqan3::dna4> text{"CCACGTCGACGGTT"_dna4};
+
+    // Here a consecutive shape with size 4 (so the k-mer size is 4) and a window size of 8 is used. The seed is set
+    // to 0, so lexicographical ordering is used for demonstration purposes.
+    auto minimisers = text | seqan3::views::minimiser_hash(seqan3::shape{seqan3::ungapped{4}}, 8, 0);
+    seqan3::debug_stream << minimisers << '\n'; // [27, 97, 26] representing the k-mers [ACGT, CGAC, ACGG]
+
+    // Here a gapped shape with size 5 (so the k-mer size is 5) and a window size of 8 is used. The seed is set
```

Comment:
Mhm!
The gapped shape size is 5, its weight (or whatever we want to call it) is 3.
So the actually generated k-mers are of size 3 (CAG, CCT, etc.).